staging/src/k8s.io/apimachinery/pkg/apis/config/v1alpha1/types.go

```diff
@@ -19,14 +19,14 @@ package v1alpha1
 // ClientConnectionConfiguration contains details for constructing a client.
 type ClientConnectionConfiguration struct {
 	// kubeConfigFile is the path to a kubeconfig file.
-	KubeConfigFile string `json:"kubeconfig"`
+	KubeConfigFile string `json:"kubeConfigFile"`
```

Comment:
I didn't expect this change. If we want to make the field consistent, I'd expect us to adjust the go struct field name

Reply:
Fair, I'll do this in a separate PR. The json tag is just wrong, so fixed it up here but will defer that change

Reply:
> The json tag is just wrong

I think the json tag is actually what we want, and the field name is wrong

Reply:
Ok, I don't have any opinions on KubeConfigFile vs KubeConfig, I'll just make it consistent to `KubeConfig` in the next PR then